tests/integration/cmd_link_unlink_test.go

```diff
@@ -68,7 +68,7 @@ var _ = Describe("odo link and unlink command tests", func() {
 			helper.CopyExample(filepath.Join("source", "python"), backendContext)
 			helper.CmdShouldPass("odo", "create", "python", "backend", "--context", backendContext, "--project", project)
 			helper.CmdShouldPass("odo", "push", "--context", backendContext)
-			stdErr := helper.CmdShouldFail("odo", "link", "backend", "--component", "frontend", "--project", project, "--context", backendContext, "--port", "1234")
+			stdErr := helper.CmdShouldFail("odo", "link", "backend", "--context", backendContext, "--port", "1234")
```

Comment:
doesn't this link component to itself?
It links component in `backendContext` to component named `backend`.

it should  be `--context frontendContext`

Reply:
Missed this one😅

Reply:
done